Deploy/stacks/dynamic/stack-clients/src/main/java/com/cmclinnovations/stack/clients/utils/FileUtils.java

```diff
@@ -127,4 +127,13 @@ public static String fixSlashs(String path, boolean leading, boolean trailing) {
         return path;
     }
 
+    public static boolean hasFileExtension(Path file, String extension) {
+        int i = file.toString().lastIndexOf(".");
+        if (i > 0) {
+            return file.toString().substring(i + 1).equals(extension);
+        } else {
+            return false;
+        }
```

Comment:
Suggested simplification:
```suggestion
        file.toString().endsWith("." + extension);
```

Reply:
Resolved by 8060bd9